app/templates/teams.hbs

```diff
@@ -3,12 +3,12 @@
 <div class="container">
   <h1 class="text-center">The Team Behind Ember</h1>
 
-  <section class="mb-3" aria-labelledby="team-detail">
```

Comment:
I believe the `aria-labelledby` attribute is to be used in conjunction with an `id` on a header. This pattern appears commonly in the project (look for `<section>` elements).

I'm not sure why, here, `aria-labelledby="team-detail"` had been added.